### ISSUE TITLE
Modify image comparison code to work with windows elements

### ISSUE THREAD

**Issue by tyBouch** [dougnoel/sentinel#303]
The current image comparison code utilizes things such as javascript executor for background color, and background-color CSS is referenced.

This is not supported by the Windows Accessibility Layer used by winappdriver, and the methods should be modified to account for this.
